generators/ionic/resources/base/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2015",
+    "target": "es2022",
```

Comment:
This change causes `npm test` to fail. If I revert it, tests pass, but the warning comes back. 
<img width="1442" alt="Screenshot 2024-06-03 at 3 55 18 PM" src="https://github.com/jhipster/generator-jhipster-ionic/assets/17892/33e6f5a8-3d1f-4886-b02a-b11615989a60">